### PR TITLE
Register service events added to tests

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d2d9c63f83181a3f58391a971f733eceaa71effb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/services/test_service_catalogs.py [hunk 1]
```diff
@@ -134,8 +134,11 @@ def test_order_catalog_item(provider_crud, provider_key, provider_mgmt, provider
     service_catalogs = ServiceCatalogs("service_name")
     service_catalogs.order(catalog_item.catalog, catalog_item)
     flash.assert_no_errors()
-    register_event(provider_crud.get_yaml_data()['type'],
-            "vm", vm_name, ["vm_power_off_req", "vm_power_off"])
+    register_event(
+        provider_crud.get_yaml_data()['type'],
```
Comment (mfalesni) [inline cfme/tests/services/test_service_catalogs.py]: I am not sure but I thin kwe have `provider_type` fixture from testgen.

**Comment by dajoRH** [comment]
Lint report for commit 10f9e723593945f34c11289b541fa868a6289c83:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc59103131954c992cdb376fd38ce18f30ae8988:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 894e64f8098eaed22819c4a32d24a7710d0d4abd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ea2d06438ca0d8fe2b90013b76069dc6b050cfbd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**